### PR TITLE
Fix the readthedocs build

### PR DESCRIPTION
**Pull request by tbetbetbe**
- its currently broken, because a caption is required in the index.rst
  for the PDF builds.  This has been added.

Also:
- improved some of the module titles
- added the latex build the tox docs target so we know early if the
  PDF doc build is affected

Change-Id: Ia3b00224741952dff62564ef3cfade399414a243

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Google API conventions.
 
 Application code will rarely need to use most of the classes within this
 library directly, but code generated automatically from the API definition
-files in `GOOGLE APIS`_ can use services such as page streaming and request bundling to provide
+files in `Google APIS`_ can use services such as page streaming and request bundling to provide
```
Comment (geigerj) [inline README.rst]: nit: s/APIS/APIs

**Comment by geigerj** [comment]
LGTM (aside from one small nit)

File: README.rst (hunk 1 above)
Comment (tbetbetbe) [inline README.rst]: Done

**Comment by geigerj** [comment]
LGTM

**Pull request merged**